src/plain_data.rs

```diff
@@ -18,7 +18,7 @@
 use rustc_serialize::{Decoder, Encodable, Encoder};
 use xor_name::XorName;
 
-/// PlainData
+/// Plain data with a name and a value.
```

Comment:
Do you mean a Domain-Specific Language?
I added that any restrictions are left to the implementation - or should I explicitly mention DSLs?